MAVProxy/modules/mavproxy_soar.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python3
+'''
+Soaring Module
+Ryan Friedman
+
+This module displays the estimated soaring thermals on the map.
+A circle is drawn with the estimated radius from ArduSoar's estimated thermal location.
+
```

Comment:
```suggestion

Note that this module uses NAMED_VALUE_FLOAT messages from the uatopilot, a temporary measure until a proper mavlink message is decided upon.

```

... so others taking inspiration from the module can't get too upset when we break these messages...